src/main/webapp/assets/feedStyle.css

```diff
@@ -1,19 +1,17 @@
-#page-container {
+#col-container {
   display: flex;
-  flex-direction: row;
+  flex-direction: column;
   flex-wrap: wrap;
   width: 80%;
   margin-bottom: 50px;
   margin-left: 10%;
   margin-right: 10%;
 }
 
-@media screen and (max-width: 678px) {
-  #page-container {
-    margin-left: 5%;
-    margin-right: 5%;
-    width: 90%;
-  }
```

Comment:
Thanks for cleaning this up! I wasn't sure whether to break it up by element type or by media query size :)

Reply:
What exactly is this change? This falls into the part I am working on so I would appreciate some communication

Reply:
Definitely! I decided to use media query size because there was multiple element types in separate but the same media query sizes so it made sense to me to put it all together!

Reply:
@ubahl Originally, the `page-container` didn't include the title of the page so it didn't align with the rest of the page so I created the `col-container` to include the title while keeping the spacing!